public/app/types/templates.ts

```diff
@@ -0,0 +1,19 @@
+export interface Variable {
```

Comment:
This contains way to many properties (from only one sub type). Think the only basic common properties for a variable is name, type & current.